.github/workflows/kind.yml

```diff
@@ -63,8 +64,8 @@ jobs:
         path: flow-aggregator.tar
         retention-days: 1 # minimum value, in case artifact deletion by 'artifact-cleanup' job fails
 
-  test-e2e-encap:
-    name: E2e tests on a Kind cluster on Linux
+  test-e2e-encap-without-timeconsumingcases:
+    name: E2e tests without time consuming cases on a Kind cluster on Linux
```

Comment:
Could you consider my suggestion in https://github.com/antrea-io/antrea/pull/2245#discussion_r664575955?
> The test-e2e-encap is actually the most typical configuration of Antrea (every configuration is the default value), so I think maybe it's worth to run the whole test in a job completely. And instead of splitting test-e2e-encap, maybe we can skip non traffic mode irrelavant tests in other tests like test-e2e-noencap, test-e2e-hybrid. For example,
> - test-e2e-encap: runs full test
> - test-e2e-noencap: skip traffic mode irrelevant test as they have been cover by test-e2e-encap, e.g. antctl_test, security_test, supportbuddle_test
> - test-e2e-hybrid: same as above

I think it's important to keep the script run all tests by default, otherwise it would be a bit bothering for developers especially new contributors to learn the usage of the script and all its parameters. Also, from style persperctive, I think the script should be kept as generic as it was, instead of too customized for the github workflows.

The title could be adjusted a little to reflect the real purpose of the change. Otherwise it would consume new e2e contributors that they should run time consuming cases in the dedicated job even when the feature may be affected by the traffic mode. We should only skip tests when they are not relevant with the traffic modes, regardless of the time they will run.

In short, my suggestion is: 
1. keep test-e2e-encap as it was (run whole tests with everything set to default)
2. skip traffic mode irrelevant tests in test-e2e-noencap and test-e2e-hybrid by adding a "--skip" parameter.

Reply:
the original purpose is to reduce the overall e2e time, so I pick up the time-consuming cases as standalone job which can help to reduce it effectively. if we choose to run the full e2e cases, then I think there will be not much difference in e2e time duration since the time of kind workflow will depend on the longest one. let's see if there any new suggestion from  @lzhecheng @antoninbas

Reply:
after discuss with @tnqn  and @lzhecheng , we came to agreement that to keep the default encap mode with full e2e list to avoid any confusion, and add mode-irrelevant-case only as skip keyword now to skip some tests for noEncap/hybrid mode.